src/modules/carousel.js

```diff
@@ -204,8 +204,8 @@ layui.define(['jquery', 'lay'], function(exports){
     
     // 模板
     var tplArrow = $([
-      '<button class="layui-icon '+ ELEM_ARROW +'" lay-type="sub">'+ (options.anim === 'updown' ? '&#xe619;' : '&#xe603;') +'</button>',
-      '<button class="layui-icon '+ ELEM_ARROW +'" lay-type="add">'+ (options.anim === 'updown' ? '&#xe61a;' : '&#xe602;') +'</button>'
+      '<button type="button" class="layui-icon '+ ELEM_ARROW +'" lay-type="sub">'+ (options.anim === 'updown' ? '&#xe619;' : '&#xe603;') +'</button>',
+      '<button type="button" class="layui-icon '+ ELEM_ARROW +'" lay-type="add">'+ (options.anim === 'updown' ? '&#xe61a;' : '&#xe602;') +'</button>'
```

Comment:
顺带把 icon 改成 className
```suggestion
      '<button type="button" class="layui-icon '+ (options.anim === 'updown' ? 'layui-icon-up' : 'layui-icon-left') + ' ' + ELEM_ARROW +'" lay-type="sub"></button>',
      '<button type="button" class="layui-icon '+  (options.anim === 'updown' ? 'layui-icon-down' : 'layui-icon-right')  + ' ' + ELEM_ARROW +'" lay-type="add"></button>'
```